### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by UrsulaMontag**
Add routing to app with four pages. Home, Map, Create-Entry and Entries-List.
Inclusive a navigation-bar at bottom of each page.

### CODE REVIEW

File: src/components/ui/Navigation.styled.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const NavBottom = styled.article`
+	max-width: 800px;
```
Comment (CJAkkuec) [inline src/components/ui/Navigation.styled.js:4]: Is there any particular reason for this "magic number"? :)

File: src/components/ui/Container.styled.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+	max-width: 800px;
```
Comment (CJAkkuec) [inline src/components/ui/Container.styled.js:4]: Same here! :) Is there any particular reason that 800px seems to be your max-width?

File: src/components/ui/Map.styled.js [hunk 3]
```diff
@@ -1,8 +1,13 @@
 import styled from 'styled-components';
 
-export const StyledMap = styled.div`
-	height: 30rem;
-	max-width: 40rem;
+const StyledMap = styled.section`
+	margin: auto;
+	border: 2px solid var(--main-bg-color-full);
```
Comment (CJAkkuec) [inline src/components/ui/Map.styled.js:5]: What's the difference between `--main-bg-color-full` and `--main-bg-color`? Usually I'd probably want my variables to be fairly verbose - meaning: I want to be able to identify their purpose as quickly as possible. :) I'm not sure the appendix -full is very verbose. What do you think?

**Review by CJAkkuec — COMMENTED** [review]
Hey there! Looking good so far! I have a couple of minor notes regarding your styling, though. (I guess that's more a "quality assurance" concern, but I figured I'd mention it anyway.)

The color for your entry headlines is barely visible on the pale sort of grey-ish red you picked as a background color. I'm sure you've got a design idea figured out already, but perhaps you'd like to reconsider your palette a little bit? I'm assuming your app is a bit more nature-oriented, and perhaps you'd want to aim for the palette to reflect said subject. :)

File: src/components/ui/Navigation.styled.js (hunk 1 above)
Comment (UrsulaMontag) [reply src/components/ui/Navigation.styled.js:4]: Hey CJ, there is actually no reason at all for this restriction, so actually something like magic . I have copied all styles so far from previous projects together and roughly adapted them to the project. I sincerely hope to find more time for my styling by the weekend, but i will definitely correct the mistakes you marked. The color scheme will probably have to wait until the weekend :-D

**Pull request merged**